definitions/infra-prometheus-redis/dashboard.json

```diff
@@ -0,0 +1,130 @@
+{
+    "title": "Redis status",
+    "icon": "line-chart",
+    "grid_column_count": 12,
+    "filter": null,
+    "owner": {
```

Comment:
We might want to generate this section post registration (otherwise this email address will pop up over all accounts :D)

Reply:
This entire section will be added internally after commit - could you remove the owner section @kidk ?